jbpm-wb-integration/jbpm-wb-integration-client/src/main/java/org/jbpm/workbench/wi/client/workitem/ServiceTasksRepositoryListPresenter.java

```diff
@@ -104,4 +134,13 @@ public void setServiceTaskService(final Caller<ServiceTaskService> serviceTaskSe
 
     }
 
-}
\ No newline at end of file
+    public void openRemoveServiceTaskConfirmPopup(ServiceTaskSummary serviceTaskSummary) {
+        if (serviceTaskSummary.getInstalledOn().size() > 0) {
+            confirmPopup.show(constants.RemoveServiceTaskTitle(), null, null, constants.Ok(),
```

Comment:
In the video it seems this popup has two buttons (Cancel, Ok), but should have only one as there's no other option then closing the popup.

Reply:
done